### ISSUE TITLE
ESPHome possible?

### ISSUE THREAD

**Issue by Dinges28** [golles/ha-kamstrup_403#49]
### Did you read the instructions?

- [X] I have read the [README](../blob/master/README.md).

### The request

Is it possible to migrate this intergration to an esphome supported device?

I have the 66C Kamstrup working by using this project: https://github.com/ProudElm/ESPHome_multical_sensor
But today they replaced it to a Kamstrup 403.

Do you know... maybe... a way to migrate your project to an ESPHome version??

### Additional information

I do have some linux programming knowledge, so maybe we can combine the effort and maybe can get this to work..

**Comment by StevenLooman** [comment]
I have a Kamstup 602 myself, but did some research before I ended up using this (great!) integration by @golles. I came accross https://github.com/bipsendk/kamstrup-382Lx7. I'm unsure if this is in the direction you're looking for, but it seems to use the protocol this integration is using as well.

I also found https://37.97.208.91/download/17894 via a search engine. I'm completely unsure where this is from but it downloads a `__init__.py` file, which I'm guessing is something which runs on [micropython](https://github.com/micropython/micropython). Perhaps that is usable?
